docs/USAGE.md

```diff
@@ -188,6 +188,41 @@ Example:
 
     OPENVINO_TF_DISABLE_DEASSIGN_CLUSTERS="1"
 
+**OPENVINO_TF_DISABLE_TFFE:**
```

Comment:
Can we use bullets for each variable to indicate where each description starts? Currently, it's hard to distinguish where a paragraph starts.